aptos-move/framework/aptos-framework/sources/configs/staking_config.move

```diff
@@ -110,8 +157,39 @@ module aptos_framework::staking_config {
     }
 
     /// Return the reward rate.
-    public fun get_reward_rate(config: &StakingConfig): (u64, u64) {
-        (config.rewards_rate, config.rewards_rate_denominator)
+    public fun get_reward_rate(_config: &StakingConfig): (u64, u64) {
```

Comment:
This change should be gated behind a feature flag. And even if we have enabled per-second rewards rate, we can still actually return the per epoch rate here by doing the math based on per-sec rate. This would avoid breaking off chain systems that have not migrated to the new rewards rate getter function.